tests/extend-expect.test.ts

```diff
@@ -0,0 +1,42 @@
+// The matcher test files cover most of the functionality.
+// This file checks behavior that is common to all matchers
+
+import { withBrowser } from 'pleasantest';
+
+test(
+  'throws useful error if Promise is passed',
+  withBrowser(async ({ screen, utils }) => {
+    await utils.injectHTML('<h1>Hi</h1>');
+    await expect(expect(screen.getByText('Hi')).toBeVisible()).rejects
+      .toThrowErrorMatchingInlineSnapshot(`
+                  "[2mexpect([22m[31mreceived[39m[2m).toBeVisible()[22m
+
+                  [31mreceived[39m value must be an HTMLElement or an SVGElement.
+                  Received a [31mPromise[39m. Did you forget to await?"
```

Comment:
Out of curiosity, what's this syntax with the `[31m` stuff?

Reply:
Those are the terminal escape codes for the colored text

Reply:
Oh, interesting! TIL, thanks!